packages/teams-js/test/public/meeting.spec.ts

```diff
@@ -1099,4 +1099,53 @@ describe('meeting', () => {
       expect(returnedSpeakingState.isSpeakingDetected).toBe(speakingState.isSpeakingDetected);
     });
   });
+
+  describe('registerRaiseHandStateChangedHandler', () => {
+    it('should fail when called without a handler', () => {
+      expect(() => meeting.registerRaiseHandStateChangedHandler(null)).toThrowError(
+        '[registerRaiseHandStateChangedHandler] Handler cannot be null',
+      );
+    });
+
+    it('should fail when called before app is initialized', () => {
+      expect(() =>
+        meeting.registerRaiseHandStateChangedHandler(() => {
+          return;
+        }),
+      ).toThrowError('The library has not yet been initialized');
+    });
+
+    it('should successfully register a handler for when the raiseHandStateChanges', () => {
+      framelessPlatformMock.initializeWithContext(FrameContexts.sidePanel, FrameContexts.meetingStage);
```

Comment:
This isn't how the `initializeWithContext` function should be called. Can you please split out the two contexts into two separate unit tests?